<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Finder</class>
 <widget class="QWidget" name="Finder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>107</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Find:</string>
     </property>
     <property name="buddy">
      <cstring>lnFind</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lnFind">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Find...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QToolButton" name="btnFindPrev">
         <property name="text">
          <string>Prev</string>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
         <property name="arrowType">
          <enum>Qt::LeftArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnFindNext">
         <property name="text">
          <string>Next</string>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
         <property name="arrowType">
          <enum>Qt::RightArrow</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="cbSelectToFind">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QToolButton" name="btnReplace">
       <property name="text">
        <string>Replace</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QComboBox" name="cbScope">
       <item>
        <property name="text">
         <string>Current file</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Open files</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Current file and references</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="3" rowspan="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>38</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Scope:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="btnReplaceAll">
       <property name="text">
        <string>Replace all</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="5" rowspan="2">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="btnInSelection">
       <property name="text">
        <string>In selection</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Replace with:</string>
     </property>
     <property name="buddy">
      <cstring>lnFind</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lnReplace">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Replace...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>18</number>
     </property>
     <item row="0" column="0" alignment="Qt::AlignLeft|Qt::AlignVCenter">
      <widget class="QCheckBox" name="cbRegex">
       <property name="text">
        <string>Regular expressions</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" alignment="Qt::AlignLeft">
      <widget class="QCheckBox" name="cbEscape">
       <property name="text">
        <string>Escape characters (\n, \t ...)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" alignment="Qt::AlignLeft">
      <widget class="QCheckBox" name="cbCaseSensitive">
       <property name="text">
        <string>Case-sensitive</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" alignment="Qt::AlignLeft">
      <widget class="QCheckBox" name="cbWholeWord">
       <property name="text">
        <string>Whole word only</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4" alignment="Qt::AlignLeft|Qt::AlignVCenter">
      <widget class="QCheckBox" name="cbDontWrap">
       <property name="text">
        <string>Don't wrap around</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lnFind</tabstop>
  <tabstop>lnReplace</tabstop>
  <tabstop>btnFindPrev</tabstop>
  <tabstop>btnFindNext</tabstop>
  <tabstop>cbSelectToFind</tabstop>
  <tabstop>btnReplace</tabstop>
  <tabstop>btnReplaceAll</tabstop>
  <tabstop>cbScope</tabstop>
  <tabstop>cbRegex</tabstop>
  <tabstop>cbEscape</tabstop>
  <tabstop>cbCaseSensitive</tabstop>
  <tabstop>cbWholeWord</tabstop>
  <tabstop>cbDontWrap</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
